lib/auto.js

```diff
@@ -102,6 +102,7 @@ export default function (tasks, concurrency, callback) {
 
     var results = {};
     var runningTasks = 0;
+    var canceled = false;
```

Comment:
Typo should be `cancelled`

Reply:
Either spelling is correct.  I chose one 'l' for consistency.

Reply:
damn americans